ruby-challenges.rb

```diff
@@ -22,13 +54,57 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# PSEUDOCODE
+# Create a method named v_gone
+# INPUT: String
+# OUTPUT: String sans vowels
+# PROCESS: .delete is a built-in method that will work on each variable individually. Create a method named v_gone that takes the beatles_album variables as arguments for the .delete method to act on. The parameter establishing the argument is called "title"
```

Comment:
You have really good pseudo code in your Ruby Challenges, I would like to see this for your code challenges as well!